sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Resources/SubscriptionOperations.cs

```diff
@@ -55,6 +55,7 @@ internal SubscriptionOperations(OperationsBase operations, TenantResourceIdentif
         /// <param name="func"> The method to pass the internal properties to. </param>
         /// <returns> Whatever the delegate returns. </returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
+        [ForwardsClientCalls]
```

Comment:
add this to tenant / rg as well

Reply:
`TenantOperations` does not have a method `UseClientContext`, but it has `ListResources`. Should we also change it to `UseClientContext` like the other two operations? We could do this in another PR.

Reply:
And I also checked the `ManagementGroupOperations`, it does not have similar functions. But I believe in the future we will need it